sdv/data_processing/data_processor.py

```diff
@@ -1,8 +1,17 @@
 """Single table data processing."""
 
+import copy
```

Comment:
Since we are using only `deepcopy` I think we can just import `from copy import deepcopy`.